ee/codegen/src/generators/nodes/generic-node.ts

```diff
@@ -45,16 +58,54 @@ export class GenericNode extends BaseSingleFileNode<
   getNodeClassBodyStatements(): AstNode[] {
     const statements: AstNode[] = [];
     this.nodeData.attributes.forEach((attribute) => {
-      statements.push(
-        python.field({
-          name: toPythonSafeSnakeCase(attribute.name),
-          initializer: new WorkflowValueDescriptor({
-            nodeContext: this.nodeContext,
-            workflowValueDescriptor: attribute.value,
-            workflowContext: this.workflowContext,
-          }),
-        })
-      );
+      if (attribute.name === "functions") {
```

Comment:
[nit] instead of hard coding here, let's use the attribute config that retry and try node uses to discriminate custom logic